src/java.base/share/classes/java/nio/MappedMemoryUtils.java

```diff
@@ -164,4 +164,17 @@ private static long alignDown(long address, int pageSize) {
         // pageSize must be a power of 2
         return address & ~(pageSize - 1);
     }
+
+    // Returns the platform defined page size, which could be different from the one Bits.pageSize
+    // is aware of. If no implementation is provided, return Bits.pageSize instead.
+    private static int pageSize() {
+        int ps = pageSize0();
+        if (ps > 0) {
```

Comment:
If `sysconf(_SC_PAGESIZE)` returns a value different from `Bits::pageSize` on other Unix platforms, this could cause an unexpected change in behavior.

Reply:
Agreed. However, my thinking was that on systems with only one pagesize the size returned by sysconf is still the correct size to return. There is no need to pop back to the VM and trigger another call (to Bits.pageSize).

Reply:
> The risk is still non-zero.

Fair point. I'll modify it to keep the same value a before, and only change AIX.